### ISSUE TITLE
Schoolism 

### ISSUE THREAD

**Issue by Lumenlul** [streamlink/streamlink#1613]
Hi,

First time trying to use Streamlink, and I'm not very savvy to how it works so I have a few questions. 

I'm trying to use Streamlink to record some Schoolism lessons that I'm subscribed to so I can watch them locally on my computer but I'm not sure how to go about this.

In plugins (C:\Program Files (x86)\Streamlink\pkgs\streamlink\plugins) I notice there is a python script for Schoolism, but to my knowledge, you can't launch that by itself? If someone could point me in the right direction of knowing how to use it, that would be appreciated. 

I also tried to open cmd and enter in the following command manually: streamlink WEBADDRESSOFVIDEO best -o "location and file name.mp4" --schoolism-email EMAIL --schoolism-password PASSWORD 

However it didn't seem to work, although it yielded "found matching plugin schoolism for url... etc"
I would assume I'd have to use the schoolism.py plugin instead, but I'm not sure how to do so.

Thanks.

**Comment by skulblakka** [comment]
>I would assume I'd have to use the schoolism.py plugin instead

No, you were doing it right. The schoolism.py file just contains the code for streamlink to use. (basically it is just part of the whole program that is streamlink).

The command you were using looks right to me. However to help you people probably need more information. You should at least copy and paste the full log of what happens after you enter the command so people know what is happening.

**Comment by Lumenlul** [comment]
The log is:

Microsoft Windows [Version 6.3.9600]
(c) 2013 Microsoft Corporation. All rights reserved.

C:\Users\[OMITTED]>streamlink https://www.schoolism.com/watchLesson.php?type=st&id=5
06 best -o "desktop/testdownload.mp4" --schoolism-email [OMITTED] --sch
oolism-password [OMITTED]
[cli][info] Found matching plugin schoolism for URL https://www.schoolism.com/wa
tchLesson.php?type=st
[plugin.schoolism][error] An email and password are required to access Schoolism
 streams
error: No playable streams found on this URL: https://www.schoolism.com/watchLes
son.php?type=st
'id' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\[OMITTED]>

**Comment by skulblakka** [comment]
> 'id' is not recognized as an internal or external command,

The URL is causing trouble because of the _&_. Try putting it in double quotation marks
